DemoApp/DemoAppCoordinator.swift

```diff
@@ -141,6 +141,7 @@ private extension DemoAppCoordinator {
         let channelListVC = DemoChatChannelListVC.make(with: controller)
         channelListVC.demoRouter?.onLogout = onLogout
         channelListVC.selectedChannel = selectedChannel
+        channelListVC.components.isChatChannelListDefaultLoadingStatesEnabled = true
```

Comment:
I think this name is a bit confusing and complex to understand, what about:
```suggestion
        channelListVC.components.isChatChannelListStatesEnabled = true
```
or 
```suggestion
        channelListVC.components.isChatChannelListViewsStatesEnabled = true
```
We should not name it "loading" because it includes or states